new-toast-site/package.json

```diff
@@ -52,6 +55,13 @@
     }
   },
   "snowpack": {
+    "exclude": [
```

Comment:
excluded some files so that things like "fs" don't show up in the breadbox output